test/gtest/ucp/test_ucp_peer_failure.cc

```diff
@@ -98,7 +99,7 @@ void test_ucp_peer_failure::init() {
     /* Make second pair */
     create_entity(true);
     create_entity(false);
-    sender().connect(&receiver(), &GetParam().ep_params_cmn);
+    sender().connect(&receiver(), get_ep_params());
```

Comment:
which `get_ep_params` implementation will be called here? I guess some one from `test_ucp_tag` parent branch. If so, error handling will be disabled. It looks like need to add `test_ucp_peer_failure::get_ep_params`

Reply:
10x, will fix

Reply:
@evgeny-leksikov there already is `test_ucp_peer_failure::get_ep_params` - in line 19

Reply:
This is `test_ucp_peer_failure_base::get_ep_params`, another branch of multi-inheritance.

Reply:
that is why we have `using test_ucp_peer_failure_base::get_ep_params;` in line 58. 
without this, even the compiler complains about ambiguity

Reply:
ahh.. right, sorry missed it.